### PR TITLE
Sum of Linked List in Python

### PR DESCRIPTION
**Pull request by iamakkkhil**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#5260

### CODE REVIEW

File: Python/ds/Sum_of_Linked_list.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Program to calculate sum of linked list.
```
Comment (ankitaggarwal23) [inline Python/ds/Sum_of_Linked_list.py]: - Add Multi-Line Comments 
- Add a beginner-friendly description at the top

File: Python/ds/Sum_of_Linked_list.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+  
+# Initialize lists as empty by creating Linkedlist objects
+head = LinkedList() 
+  
+# Pushing elements into LinkedList
+head.push(20)
+head.push(30)
+head.push(40)
+head.push(50)
```
Comment (ankitaggarwal23) [inline Python/ds/Sum_of_Linked_list.py]: Don't use static inputs use dynamic inputs from user.

File: Python/ds/Sum_of_Linked_list.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+"""
+OUTPUT: 
+Entered Circular Linked List:
+20 30 40 50
+
+Sum of Linked List: 140
+
+
+
+Time Complexity: O(n)
```
Comment (ankitaggarwal23) [inline Python/ds/Sum_of_Linked_list.py]: Remove extra two line and add SC as well

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by ankitaggarwal23** [comment]
Resolve merge conflicts.

**Comment by iamakkkhil** [comment]
Hey, I am not getting how to remove the merge conflict can have a look at it please

**Review by ankitaggarwal23 — COMMENTED** [review]
Always do `git pull `and then do the changes because it will update your repo with the changes added.

As I 'm closing this PR looks very confusing.  Open a new and the do the changes

**Pull request closed without merging**